driver-mct/main/cime_comp_mod.F90

```diff
@@ -4766,9 +4766,21 @@ subroutine cime_run_calc_budgets1()
     ! it will also use the current r2x_ox here which is the value from the last timestep
     ! consistent with the ocean coupling
 
+    logical,intent(in),optional :: in_cplrun  ! flag indicating whether routine
+                                              ! called within the scope of the
+                                              ! CPL:RUN timer
+
+    logical :: lcplrun
+    !-------------------------------------------------------------------------------
+
+    lcplrun  = .true.
+    if (present(in_cplrun)) then
+       lcplrun = .not. in_cplrun
```

Comment:
Why set lcplrun to the not of in_cplrun?   Don't you want them to have the same logical value?

Reply:
If in_cplrun is .true., then already within the scope of a CPL:RUN timer, and don't want to start/stop another CPL:RUN timer. Would be nice to be able to query this directly, but "recursive" calls to a timer can occur in valid situations (e.g., when a function containing timers  is called recursively). CPL:RUN is a different sort of timer, and these recursive calls are a distraction.